# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1373310634 -7200
#      Mon Jul 08 21:10:34 2013 +0200
# Node ID 5c25f7ed080c549183b61c0806f211baa0b5ac3e
# Parent  aebb54d99dbaee42d1ec0be373e4b575a9e76d39
*__magick_read__.cc: add missing include of gripes.h

diff --git a/libinterp/dldfcn/__magick_read__.cc b/libinterp/dldfcn/__magick_read__.cc
--- a/libinterp/dldfcn/__magick_read__.cc
+++ b/libinterp/dldfcn/__magick_read__.cc
@@ -31,16 +31,18 @@ along with Octave; see the file COPYING.
 #include "file-stat.h"
 #include "oct-env.h"
 #include "oct-time.h"
 
 #include "defun-dld.h"
 #include "error.h"
 #include "ov-struct.h"
 
+#include "gripes.h"
+
 #ifdef HAVE_MAGICK
 
 #include <Magick++.h>
 #include <clocale>
 
 octave_value_list
 read_indexed_images (std::vector<Magick::Image>& imvec,
                      const Array<int>& frameidx, bool wantalpha)
